docs/user-documentation/file_viewers.md

```diff
@@ -0,0 +1,84 @@
+# File Viewers
+
+_last updated on 03-12-2020_
+
+
+## What are viewers?
+
+[Viewers](../glossary#viewer) allow site builders to display files in interactive javascript-based widgets, that provide functionality like zooming in/out, turning pages, playing/pausing, viewing in full screen, etc. 
+
+In Drupal, a common way to implement a viewer is through a [module](./glossary/#module) that provides the javascript library, and a field formatter that uses that library. Usually the field formatter can be used on File fields.  
+
+Viewers that are known to work with Islandora 8 include:
+
+* [OpenSeadragon](https://openseadragon.github.io/), via the Drupal module [OpenSeadragon](https://github.com/Islandora/openseadragon) (maintained by the Islandora Foundation)
+* [pdf.js](https://github.com/mozilla/pdf.js), via the Drupal contrib module [PDF](https://www.drupal.org/project/pdf)
+
+Both are included with Islandora Defaults.
+
+## How are viewers configured?
+
+In the generic Islandora 8 content modelling scenario, we often want the viewer to appear on a _"resource node"_ when the file itself is in a field on a Media that references that node. These layers of indirection make displaying viewers on nodes rather complex.
+
+## Example - Open Seadragon (in Islandora Defaults)
+
+Here is how viewers are implemented in Islandora Defaults. This example uses OpenSeadragon, but pdf.js is configured with the same method. This method was chosen because it uses pure configuration and no extra code. It relies on users tagging nodes with appropriate "display hint" terms to trigger Contexts to switch the node's view mode to one that includes an EVA field, which renders the appropriate Media's file in the viewer.
```

Comment:
This might be a bit much, but there is a few more steps in here. "... which finds the right Media and sets the Media display mode which is configured to use the desired file field formatter (viewer)."

Reply:
This is a high-level overview, and those steps are outlined below. 

"finds the right media" is implied by "the appropriate media" (since the EVA does not actually determine which media is appropriate in terms of media use - that decision is configured in the node display mode configuration, by which of the three available EVAs is made visible)

What about "which renders the appropriate Media to show the file, formatted through the viewer"? 
I want to use "render" rather than "set display mode" beacuse rendering-an-entity-in-views-where-you-choose-a-display-mode, and "setting display modes by putting terms on them and creating contexts with reactions that cause the display mode to be used".  Those are two ways of "setting a display mode" (and, hilariously, sometimes clobber each other).

Reply:
Sure, "which renders the appropriate Media to show the file, formatted through the viewer" works for me.